packages/webhook-utils/src/recreate-webhooks.ts

```diff
@@ -0,0 +1,37 @@
+import { Client } from "urql";
+import { getAppDetailsAndWebhooksData } from "./operations/get-app-details-and-webhooks-data";
+import { removeAppWebhook } from "./operations/remove-app-webhook";
+import { WebhookManifest } from "@saleor/app-sdk/types";
+import { createAppWebhook } from "./operations/create-app-webhook";
+import { WebhookEventTypeAsyncEnum, WebhookEventTypeSyncEnum } from "../generated/graphql";
+
+interface recreateWebhooksArgs {
+  client: Client;
+  webhookManifests: Array<WebhookManifest>;
+}
+
+export const recreateWebhooks = async ({ client, webhookManifests }: recreateWebhooksArgs) => {
+  const appData = await getAppDetailsAndWebhooksData({ client });
+  const webhooks = appData?.webhooks || [];
+
+  // Remove all existing webhooks
+  const removeAllWebhooksPromise = await Promise.all(
+    webhooks.map((webhook) => removeAppWebhook({ client, webhookId: webhook.id })),
+  );
+
+  const createAllWebhooksPromise = await Promise.all(
+    webhookManifests.map((webhookManifest) =>
+      createAppWebhook({
+        client,
+        input: {
+          asyncEvents: webhookManifest.asyncEvents as WebhookEventTypeAsyncEnum[],
+          syncEvents: webhookManifest.syncEvents as WebhookEventTypeSyncEnum[],
+          isActive: webhookManifest.isActive,
+          name: webhookManifest.name,
+          targetUrl: webhookManifest.targetUrl,
+          query: webhookManifest.query,
+        },
+      }),
+    ),
+  );
+};
```

Comment:
these are not transactional, it can fail during the migration, eg half will be deleted

can we make it more robust? e.g:
1. Start with creating new webhooks, then delete old ones?
2. error handling -> retry policy, if migration fails, let EACH mutation to try e.g 5 times